src/bp-integrations/learndash/library/SyncGenerator.php

```diff
@@ -845,7 +845,7 @@ public function updateBuddypressGroup( $ld_group_id, $groupId ) {
 					'group_id'    => $groupId,
 					'creator_id'  => $ldGroup->post_author,
 					'name'        => $ldGroup->post_title ?: "For Social Group: {$this->ldGroupId}",
-					'status'      => $settings->get( 'learndash.default_bp_privacy' ),
+					//'status'      => $settings->get( 'learndash.default_bp_privacy' ),
```

Comment:
@mahedihasannoman I don't understand why we remove status sync.

Reply:
Hi @dipskakadiya , It is not syncing status from LD group, It is adding the default status from sync settings for every LD group update. See the screenshot: https://nimb.ws/LiCa2l
and this is the issue that have been reported here.

Also according to the settings label (marked on the screenshot) this status should be assigned upon group creation only.